src/main/java/team/chisel/client/handler/BlockSpeedHandler.java

```diff
@@ -34,7 +36,11 @@ public static void speedupPlayer(PlayerTickEvent event) {
                 manualInputCheck = new MovementInputFromOptions(Minecraft.getMinecraft().gameSettings);
             }
             EntityPlayerSP player = (EntityPlayerSP) event.player;
-            IBlockState below = player.getEntityWorld().getBlockState(player.getPosition().down());
+            BlockPos pos = new BlockPos(
+                Math.floor(player.posX),
+                Math.floor(player.posY),
+                Math.floor(player.posZ));
+            IBlockState below = player.getEntityWorld().getBlockState(pos.down());
```

Comment:
`.down()` can be excluded in favour of `player.posY - 1` in the `pos` constructor parameters, to avoid an additional `BlockPos` being constructed each time.